scripts/prepare-badger-integration-tests.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
doesn't look like this script is called from anywhere

Reply:
it is called form the `badger_test.go`

Reply:
Oh, in this case could we not do this in Go?